docker-for-mac/release-notes.md

```diff
@@ -13,7 +13,54 @@ This page contains information about the new features, improvements, known issue
 For information about Edge releases, see the [Edge release notes](edge-release-notes). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
-## Stable Releases of 2019
```

Comment:
You've deleted "Stable Releases of 2019" but not the other years further down the page. Was this intentional or accidental?

Reply:
Yes, this is an intentional update to fix some formatting issues. Both headings were at the same level, and therefore the subsections did not render correctly on the mini table of contents on the right-hand side. This is an attempt toward simplifying this issue. This is also consistent with the format used in Edge release notes. When we do subsequent releases in 2020, we can collapse the 2019 entries and ensure only new versions are expanded in the default view.